src/sage/combinat/q_analogues.py

```diff
@@ -866,7 +877,7 @@ def q_stirling_number1(n, k, q=None):
         q^3 + 3*q^2 + 4*q + 3
 
         sage: all(stirling_number1(6,k) == q_stirling_number1(6,k)(1)
-        ....:     for k in range(1,7))
+        ....:     for k in range(1,6))
```

Comment:
Why this change?

Reply:
this was slightly slow and rather superfluous